test/integ_tests/test_ahs_device.py

```diff
@@ -157,3 +162,27 @@ def circuit():
             return qml.sample()
 
         circuit()
+
+    @pytest.mark.parametrize("H, params", HAMILTONIANS_AND_PARAMS)
+    @pytest.mark.parametrize(
+        "local_detuning, local_params, local_wires", [(amp, [[0.5, 1.1, 2.9]], [0, 1]), (4.5, [], [1, 2])]
+    )
+    def test_circuit_can_be_called_local_detunings(
+        self, H, params, local_detuning, local_params, local_wires
+    ):
+        """Test that the circuit consisting of a ParametrizedEvolution with a one global pulse as well
+        as local detunings runs successfully for combinations of amplitude, phase, local detuning being
+        constants or callables."""
+
+        dev = qml.device("braket.local.ahs", wires=3)
+        H += rydberg_drive(0, 0, local_detuning, local_wires)
+        params += local_params
+
+        t = 1.13
+
+        @qml.qnode(dev)
+        def circuit():
+            ParametrizedEvolution(H, params, t)
+            return qml.sample()
+
+        circuit()
```

Comment:
anything we could assert here? I guess currently this is just implicitly_ checking that no error raised

Reply:
Not really. We can check if the output is valid, though that might be better for a different test.